optuna/importance/_base.py

```diff
@@ -132,3 +137,51 @@ def _check_evaluate_args(completed_trials: List[FrozenTrial], params: Optional[L
                     "Study must contain completed trials with all specified parameters. "
                     "Specified parameters: {}.".format(params)
                 )
+
+
+def _split_distributions(
```

Comment:
Personaly, `_split_distributions` sounds vague. How about the following name?
```suggestion
def _split_distributions_to_single_and_nonsingle_distributions(
```
or `_split_distributions_to_single_and_nonsingle`.